rudder-server-relay/SPECS/rudder-server-relay.spec

```diff
@@ -234,7 +259,7 @@ service %{apache} start > /dev/null && echo " Done"
 /bin/systemctl start  %{apache}.service && echo " Done"
 %endif
 
-
+# TODO : do not display this on webapp
```

Comment:
Add this information in the comment: if this is not the root server you have to ...